widgets/common/src/UpdateDeploymentModal.tsx

```diff
@@ -220,7 +220,11 @@ function UpdateDeploymentModal({ open, deploymentId, deploymentName, onHide, too
     return (
         <Modal open={open} onClose={onHide} className="updateDeploymentModal">
             <Modal.Header>
-                <Icon name="edit" /> Update deployment {deploymentName} ({deploymentId})
+                <Icon name="edit" />{' '}
+                {`Update deployment ${Stage.Utils.formatDisplayName({
+                    id: deploymentId,
+                    displayName: deploymentName
+                })}`}
```

Comment:
Minor:

I don't think we need to make this be a template literal :smile: 

```suggestion
                Update deployment {Stage.Utils.formatDisplayName({
                    id: deploymentId,
                    displayName: deploymentName
                })}
```